README.md

```diff
@@ -107,7 +107,20 @@ Rest Server uses exactly the same directory structure as local backend, so you s
 
 ### Systemd
 
-There's an example [systemd service file](https://github.com/restic/rest-server/blob/master/examples/systemd/rest-server.service) included with the source, so you can get Rest Server up & running as a proper Systemd service in no time.  Before installing, adapt paths and options to your environment.
+There's an example [systemd service file][systemd-service-file] included with the source, so you can get Rest Server up & running as a proper Systemd service in no time.
+
+The service file can directly be copied (typically in `/usr/lib/systemd/system`) and customised before starting the service with for example:
```

Comment:
The purpose of the `edit` operation is to have that derivate file in `/etc/systemd/system`.

Again, I guess the issue is that there are multiple kind of users; I was in the perspective where `rest-server` was packaged with a default `rest-server.service` deployed by the packager, so I was making the user re-create a similar process: deploy a default and customize into `/etc`.

Reply:
I don't really see the point in installing the example service, which doesn't work without modifications, into a directory that is supposed to be exclusively managed by the package manager and then completely overwriting that configuration file.

Or from a slightly different perspective, after the call to `systemctl edit --full` one could just as well remove the service from `/usr/lib/systemd/system` again. And that makes me question why it should be installed there in the first place.

Reply:
The current example systemd unit should probably remain just that: an example that you edit to match your configuration before you install it.

Once we have a config file, it will be easier to move all of this configuration to a standard location and distribute it as standard rpms & debs.